src/commands/git-metadata/library.ts

```diff
@@ -33,21 +37,41 @@ export const getGitCommitInfo = async (filterAndFormatGitRepoUrl = true): Promis
   return [gitRemote ?? '', payload.hash]
 }
 
-// UploadGitCommitHash uploads local git metadata and returns the current [repositoryURL, commitHash].
+// uploadGitCommitHash uploads local git metadata and returns the current [repositoryURL, commitHash].
```

Comment:
why are you lowercasing the first letter? was it wrong?

Reply:
It's just the comment, so I'm updating it to match the function name :)